src/sqltest/parts/query/editor/queryEditor.test.ts

```diff
@@ -30,28 +26,13 @@ import { TestStorageService } from 'vs/workbench/test/workbenchTestServices';
 
 suite('SQL QueryEditor Tests', () => {
 	let instantiationService: TypeMoq.Mock<InstantiationService>;
```

Comment:
Could you add some tests for the new functionality your adding such as the context key additions?

Reply:
I removed the context key stuff, There isn't any new functionality here, but I will update the action tests to fully check the state object.